### PR TITLE
[STY] shellcheck and pep8 conform restingStatePreprocess.sh

### PR DESCRIPTION
**Pull request by jdkent**
Changes proposed in this pull request
- shellcheck and pep8 conform

### CODE REVIEW

File: restingStatePreprocess.sh [hunk 1]
```diff
@@ -89,36 +89,36 @@ if [ "$FSLDIR" == "" ]; then
 fi
 
 
-  #A few default parameters (if input not specified, these parameters are assumed)
-  if [[ $te == "" ]]; then
-    te=30
-  fi
+#A few default parameters (if input not specified, these parameters are assumed)
```
Comment (mwvoss) [inline restingStatePreprocess.sh]: pep8 missed space

File: restingStatePreprocess.sh [hunk 2]
```diff
@@ -225,69 +225,62 @@ if [[ -e $indir/${preprocfeat} ]]; then
...
+    epiWarpDir=$(dirname $epiWarpDirtmp)
 
-      #T1toMNI
-      T1WarpDirtmp=`cat $indir/rsParams | grep "MNItoT1IWarp=" | tail -1 | awk -F"=" '{print $2}'`
-        T1WarpDir=`dirname $T1WarpDirtmp`
+    if [[ $fieldMapFlag == 1 ]]; then
+      #Copy the EPItoT1 warp file
+      cp  $epiWarpDir/EPItoT1_warp.nii.gz $regDir/example_func2highres_warp.nii.gz
+    else
+      #Only copy the affine .mat files
```
Comment (mwvoss) [inline restingStatePreprocess.sh]: pep8 spacing

**Comment by mwvoss** [comment]
saw a couple pep8 spacing misses, just for consistency flagged them with a single line comment

**Comment by jdkent** [comment]
good catches @mwvoss, fixed those.

**Comment by jdkent** [comment]
going to merge this in now.

**Pull request merged**